plugin/pkg/scheduler/algorithm/predicates/predicates.go

```
@@ -1157,24 +1169,24 @@ func (c *PodAffinityChecker) satisfiesExistingPodsAntiAffinity(pod *v1.Pod, meta
 		filteredPods, err := c.podLister.FilteredList(nodeInfo.Filter, labels.Everything())
 		if err != nil {
 			glog.Errorf("Failed to get all pods, %+v", err)
-			return false
+			return false, nil
```

Please return an error here as well.